Trappist/src/Promact.Trappist.Repository/BasicSetup/BasicSetupRepository.cs

```diff
@@ -136,7 +113,7 @@ private string GetConnectionString(SqlConnectionStringBuilder connectionString)
         /// </summary>
         /// <param name="model"></param>
         /// <returns>It return true or false</returns>
-        private bool SaveSetupParameter(BasicSetupModel model)
+        private bool isSetupParameterSavedSuccessfully(BasicSetupModel model)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Rename method "isSetupParameterSavedSuccessfully" to match camel case naming rules, consider using "IsSetupParameterSavedSuccessfully". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS100)